lib/openapi3.js

```diff
@@ -44,18 +44,18 @@ function convertToToc(source,data) {
                     sMethodUniqueName = method.operation.summary;
                 }
                 method.slug = sMethodUniqueName.toLowerCase().split(' ').join('-'); // TODO {, } and : ?
-                var tagName = data.translations.defaultTag;
-                var tagDescription = '';
-                if (method.operation.tags && method.operation.tags.length > 0) {
-                    var tagData = getTagGroup(method.operation.tags[0], data.options.tagGroups);
-                    tagName = tagData.name;
-                    tagDescription = tagData.description;
+                if (typeof method.operation.tags === 'undefined' || !method.operation.tags.length) {
+                    method.operation.tags = [data.translations.defaultTag];
                 }
-                if (!resources[tagName]) {
-                    resources[tagName] = { count: 0, methods: {}, description: tagDescription};
-                }
-                resources[tagName].count++;
-                resources[tagName].methods[sMethodUniqueName] = method;
+                method.operation.tags.forEach(function(tag){
```

Comment:
Informational: from a purely syntactic standpoint, `function () {}` means `function <anonymous>()`; `function doSth() {}` means `function with name doSth`; `var doSth = function () {}` means `create anonymous function and assign it to var doSth`.  In the current case note that `function() {}` creates a conflict between anonymous and named (with reserved word `function`).

More to the point, if an error occurs, anonymous functions are a pain, so naming things helps (`forEach(function addTag(tag) {}`).